src/main/java/bisq/core/dao/node/parser/OpReturnBlindVoteParser.java

```diff
@@ -44,6 +44,7 @@ public OpReturnBlindVoteParser(PeriodService periodService,
     // We do not check the version as if we upgrade the a new version old clients would fail. Rather we need to make
     // a change backward compatible so that new clients can handle both versions and old clients are tolerant.
     boolean validate(byte[] opReturnData, long bsqFee, int blockHeight, ParsingModel parsingModel) {
+        //TODO remove
```

Comment:
Some of these `TODO`s seems like they would be easy to address in this PR, but in discussion with @ManfredKarrer he prefers to merge as-is and send follow-up PRs to remove these now unnecessary `OP_RETURN` helper classes.